### DISCUSSION TITLE
Running a GWT sim with a subset of kper from GWF

### DISCUSSION THREAD

**Discussion by p-ortega** [MODFLOW-ORG/modflow6#1416] [Q&A] [answered]
Hi! I'm running a GWT sim outside of the main GWF simulation. 

I'm saving all flows and heads for all stress periods from the GWF, but I'd like to run the GWT with a subset of those stress periods, let's say for example the last 10 stress periods. 

I tried running the GWT with a shorter list of stress periods but runs from kper = 0 and I'd like to start at kper > 0.

There is an easy way to do this?

Cheers,
Pabs

**Comment by christianlangevin** [comment, marked as answer]
Hi @p-ortega, sadly there is not a quick and easy way to do this directly.  I think your only option would be to read the head and budget files and rewrite them with the intended information with kstp/kper values set accordingly.  There are utilities in flopy that can help with this, but it would take a little effort.

**Comment by p-ortega** [comment]
Hi @langevin-usgs , thanks for the quick response! I had a feeling this might be the case. I've already had a workaround in mind, but I just wanted to double-check before diving down the rabbit hole. 😄

Thank you again!
